### DISCUSSION TITLE
How do i create a composed chart

### DISCUSSION THREAD

**Discussion by plsmarann** [patrykandpatrick/vico#790] [Questions]
### Question

I need to create a chat with bar and a line chart in one .

### Vico version(s)

v 1.14.0
### UI framework(s)

Compose

**Comment by Gowsky** [comment]
Hello, @plsmarann. Please see [Handling data](https://www.patrykandpatrick.com/vico/guide/1.16.0/handling-data) and [`ComposedChart`](https://www.patrykandpatrick.com/vico/guide/1.16.0/charts#composedchart) in the guide.
